internal/renderer/service_util.go

```diff
@@ -356,6 +356,13 @@ func createLbService4Gateway(c *RenderContext, gw *gwapiv1a2.Gateway) *corev1.Se
 		})
 	}
 
+	healthCheckPort, err := setHealthCheck(as, svc)
+	if err != nil {
+		c.log.V(1).Info("%s", err)
+	} else {
```

Comment:
We'll have to distinguish 3 cases:
- there is no HC annotation and no error occurred in `setHealthCheck` (in this case `err==nil` and `healthCheckPort==0`),
- there was a HC annotation but something went wrong (in this case `err!=nil` and `healthCheckPort==0`), or
- there was  a HC annotation and everything went fine (in this case `err==nil` and `healthCheckPort!=0`). 

The first case should not report anything, the second case should report the error, and the third one should report "HC port opened". So let's change the if-then-else this way:
```
healthCheckPort, err := setHealthCheck(as, svc)
if err != nil {
   c.log.V(1).Info("%s", err)
} else if healthCheckPort != 0 {
   c.log.V(1).Info("Health Check port %d opened", healthCheckPort)
}
```